.babelrc

```diff
@@ -0,0 +1,11 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "browsers": ["last 2 versions"]
+      },
+      "modules": false
+    }]
+  ],
+  "plugins": ["external-helpers"]
```

Comment:
Are we using `external-helpers`? From what I understand here (https://babeljs.io/docs/plugins/external-helpers/) is they they have to be explicitly built and included separately. It doesn't seem like we do it. Also doesn't seem like we would need to do it in a short-term (as our codebase is not too big enough to benefit from such optimisations).

Unless rollout does something with that automatically?